plugin/src/utils/cache.ts

```diff
@@ -125,7 +142,11 @@ export const setHardCachedData = async ({ key, value }) => {
   await hardCache.set(key, value)
 }
 
-export const getHardCachedData = async ({ key }) => {
+export const getHardCachedData = async <T = Node>({
```

Comment:
This isn't necessarily a node but can be any cacheable data (so a string, number, object, node, array, etc)

Reply:
yeah, i make it a gatsby node by default because most of the cases were for returning `Node[]` or `node`, but the type arg is overridable, so you can provide whatever response type when casting